testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -673,7 +674,62 @@ public void testMaterializedView()
 
         assertQueryReturnsEmptyResult(listMaterializedViewsSql("name = '" + view.getObjectName() + "'"));
         assertQueryReturnsEmptyResult(listMaterializedViewsSql("name = '" + otherView.getObjectName() + "'"));
-        assertQueryReturnsEmptyResult(listMaterializedViewsSql("name = '" + viewWithComment.getSchemaName() + "'"));
+        assertQueryReturnsEmptyResult(listMaterializedViewsSql("name = '" + viewWithComment.getObjectName() + "'"));
+    }
+
+    @Test
+    public void testRenameMaterializedView()
```

Comment:
I don't think those two tests influence each other. I executed 30+ runs of only those two in parallel with 100% success rate.
btw. Given how big is `testMaterializedView` already I would lean toward removing the flakiness if it existed instead of extending `testMaterializedView`.